curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/64440a50d076c204051f2cc0.md

```diff
@@ -0,0 +1,212 @@
+---
+id: 64440a50d076c204051f2cc0
+title: Step 55
+challengeType: 0
+dashedName: step-55
+---
+
+# --description--
+
+You need a function that will be used to convert view counts to a more readable format. For example, if the view count is 1000, it should be converted to 1k. If the view count is 100,000 then it will be 100k. For that, create a function named `countingViews`
```

Comment:
```suggestion
You need a function to convert view counts to a more readable format. For example, if the view count is 1000, it should be converted to 1k. If the view count is 100,000 then it will be 100k. For that, create a function named `countingViews`
```

Reply:
added